<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicGame</class>
 <widget class="QWidget" name="BasicGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>131</width>
     <height>340</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>  积分：</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreLabel">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="startGameBtn">
      <property name="maximumSize">
       <size>
        <width>169</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseGameBtn">
      <property name="text">
       <string>暂停游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hitTextBtn">
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="restartGameBtn">
      <property name="text">
       <string>重新开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rescheduleBtn">
      <property name="text">
       <string>重排</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endGameBtn">
      <property name="text">
       <string>结束游戏</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">border:1px solid #FFFFFF;
height:30;

text-align: center; 
color:rgb(28, 161, 4);
border-radius:10px; 

      
</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">clor:rgb(255, 103, 103);
font: 16pt &quot;Adobe 黑体 Std R&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>158</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="settingBtn">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="helpBtn">
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
